webpack.config.js

```diff
@@ -16,12 +17,12 @@ module.exports = {
         }
     },
     module: {
-        loaders: [
+        rules: [
             {
                 test: /\.js$/,
                 loader: 'babel-loader',
                 query: {
-                    presets: ['env', 'stage-0'],
+                    presets: ['@babel/preset-env'],
```

Comment:
I left it like this but is there a reason that we don't use the `.babelrc` config ? Each webpack file has it's own babel setup, seems that it could just use one